src/lib/config/ModelConfig.cc

```diff
@@ -370,8 +370,7 @@ void ModelConfig::setExpectations(bool isDefault) {
     expectations_["Core"].addChild(ExpectationNode::createExpectation<uint64_t>(
```

Comment:
If we know that the maximum value for this is 2048, do we need to use unit64_t?  Elsewhere we have gone for the smallest suitable datatype.  Not sure if there is a technical advantage to using a smaller int type, but if there is then it might apply here as well.

Reply:
Good point / spot - changing to uint16_t